public/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 3,
   "name": "UBC Workday Side by Side Calendar",
-  "version": "1.5.1",
+  "version": "1.6.0",
```

Comment:
can you update the firefox manifest as well?

Reply:
sure

Reply:
wait where is the firefox manifest?